src/App/Expression.cpp

```diff
@@ -168,6 +169,12 @@ static inline T &&cast(App::any &&value) {
 #endif
 }
 
+static inline bool cast_bool(double value)
+{
+    return std::fabs(value) >= Base::Precision::Confusion();
+}
+
+
```

Comment:
Consider putting this in an anonymous namespace instead of using static: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities

The function isn't really casting the value so maybe "toBool(double value)" would be a more describing name?